### PR TITLE
feat(molecule/fieldTextarea): component moleculeFieldTextarea

### PR DESCRIPTION
**Pull request by juanmaguitar**

### CODE REVIEW

**Comment by juanmaguitar** [comment]
Online demo can be checked at → https://sui-components-feature-molecule-field-textarea.now.sh/workbench/molecule/fieldTextarea/demo

**Review by jcger — APPROVED** [review]

File: demo/molecule/fieldTextarea/playground [hunk 1]
```diff
@@ -0,0 +1,62 @@
+const styleList = {
```
Comment (jcger) [inline demo/molecule/fieldTextarea/playground:1]: you could add one example with less `maxCharacters` in order to test it easier

File: components/molecule/fieldTextarea/README.md [hunk 2]
```diff
@@ -0,0 +1,93 @@
...
+```
+
+### With maxCharacters
+    
+```
+  <MoleculeFieldTextarea
+    id="description"
+    label="Description"
+    value="In some place of La Mancha which name..."
+    maxCharacters={75}
```
Comment (davidbarna) [inline components/molecule/fieldTextarea/README.md]: the shorter the API the better as long as it's easily understandable. IMO `maxChars` would be a better name.

**Review by davidbarna — APPROVED** [review]

File: components/molecule/fieldTextarea/README.md (hunk 2 above)
Comment (juanmaguitar) [reply components/molecule/fieldTextarea/README.md]: 👍 changed to `maxChars`

File: demo/molecule/fieldTextarea/playground (hunk 1 above)
Comment (juanmaguitar) [reply demo/molecule/fieldTextarea/playground:1]: 👍 `maxChars={30}`

**Pull request merged**